src/observable/throw.ts

```diff
@@ -4,7 +4,7 @@ import {Subscription} from '../Subscription';
 
 export class ErrorObservable<T> extends Observable<T> {
 
-  static create<T>(error: T, scheduler?: Scheduler) {
+  static create<T>(error: any, scheduler?: Scheduler) {
```

Comment:
I am currently trying on some code, it seem static method can not use Type information in static methods.
Namely, 
```typescript
class ErrorObservable<T> extends Observable<T> {
  static create(value: any): ErrorObservable<T> {
    return new ErrorObservable<T>(); // Compile time error for name T.
  }
}
``` 
would be illegal, and must be
```typescript
class ErrorObservable<T> extends Observable<T> {
  static create<T>(value: any): ErrorObservable<T> {
    return new ErrorObservable<T>();
  }
}
```
And now I have
```typescript
class Usage {
  getItem(id: string): Observable<Item> {
    // if found
    return Observable.of(item);
    // if not found
    return Observable.throw(new Error('...')); // It's illegal, can not cast ErrorObservable to Observable<Item>.
    // then
    return Observable.throw<Item>(new Error('...')); // it would be fine.
  }
}
```